src/components/account.js

```diff
@@ -1,14 +1,18 @@
-import React from 'react';
+import React, { useContext } from 'react';
 
 import { Header, Footer } from './header-footer'
 
-export default function Account(props) {
-   const { user } = props
+//State Management - Context API
+import { UserContext } from '../contextAPI';
+
+export default function Account() {
+   const user = useContext(UserContext)
```

Comment:
Nit: might want to change this to userContext so when you destructure it doesn't look as odd as user.user.first_name